Sources/CreateAPI/Generate.swift

```diff
@@ -20,8 +20,11 @@ struct Generate: ParsableCommand {
     @Option(help: "The path to generator configuration. If not present, the command will look for .create-api.yaml in the current directory.")
     var config = "./.create-api.yaml"
 
-    @Flag(name: .shortAndLong, help: "Split output into separate files")
-    var split = false
+    @Flag(help: ArgumentHelp("Merge Entities and Paths into single output files", discussion: """
+                            Merging the source files offers a compact output, but prevents the compiler \
+                            from parallelizing build tasks resulting in slower builds for larger schemas.
+                            """))
```

Comment:
Should only be one level of indentation here

```suggestion
    @Flag(help: ArgumentHelp("Merge Entities and Paths into single output files", discussion: """
        Merging the source files offers a compact output, but prevents the compiler \
        from parallelizing build tasks resulting in slower builds for larger schemas.
        """))
```